prow/plugins/approve/approvers/approvers_test.go

```diff
@@ -908,3 +908,48 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 		t.Errorf("GetMessage() = %+v, want = %+v", *got, want)
 	}
 }
+
+func TestGetMessageMDOwners(t *testing.T) {
+	ap := NewApprovers(
+		Owners{
+			filenames: []string{"a/a.go", "b/README.md"},
+			repo: createFakeRepo(map[string]sets.String{
+				"a":           sets.NewString("Alice"),
+				"b":           sets.NewString("Bill"),
+				"b/README.md": sets.NewString("Doctor"),
+			}),
+			log: logrus.WithField("plugin", "some_plugin"),
+		},
+	)
+	ap.AddAuthorSelfApprover("John", "REFERENCE", false)
+	ap.RequireIssue = true
+	want := `[APPROVALNOTIFIER] This PR is **NOT APPROVED**
+
+This pull-request has been approved by: *<a href="REFERENCE" title="Author self-approved">John</a>*
+To fully approve this pull request, please assign additional approvers.
+We suggest the following additional approvers: **alice**, **doctor**
+
+Assign the PR to them by writing ` + "`/assign @alice @doctor`" + ` in a comment when ready.
+
+*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with ` + "`/approve no-issue`" + `
+
+The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).
+
+The pull request process is described [here](https://git.k8s.io/community/contributors/devel/owners.md#the-code-review-process)
+
+<details open>
+Needs approval from an approver in each of these OWNERS Files:
```

Comment:
This line might be confusing if the below list contains `.md` files. We can make it more clear I guess.

Reply:
Fixed.